hadoop-ozone/ozonefs/src/main/java/org/apache/hadoop/fs/ozone/OFSPath.java

```diff
@@ -67,10 +75,9 @@ private void initOFSPath(String pathStr) {
       // TODO: Compare a keyword list instead for future expansion.
       if (firstToken.equals(OFS_MOUNT_NAME_TMP)) {
         mountName = firstToken;
-        // TODO: Retrieve volume and bucket of the mount from user protobuf.
-        //  Leave them hard-coded just for now. Will be addressed in HDDS-2929
-        volumeName = "tempVolume";
-        bucketName = "tempBucket";
+        // Future: retrieve volume and bucket from UserVolumeInfo
```

Comment:
Do you mean to comment a TODO here?

Reply:
Done in 5ed5cfa523cb6a0ff1d2c5179ef3f0f04d8c7ae9